src/tests/unit/tests/electron/platform/android/appium-service-configurator.test.ts

```diff
@@ -200,13 +213,13 @@ describe('AppiumServiceConfigurator tests', () => {
         expect(info.versionName).toBeUndefined();
 
         adbMock.verifyAll();
+        apkLocatorMock.verifyAll();
     });
 
     it('getPermissionInfo, propagates error', async () => {
-        const expectedMessage = 'Something bad happened';
-        adbMock.setup(m => m.setDeviceId(emulatorId)).verifiable(Times.once());
+        const expectedMessage = 'Thrown during getPermissionInfo';
         adbMock
-            .setup(m => m.shell(['dumpsys', 'accessibility']))
+            .setup(m => m.setDeviceId(emulatorId))
```

Comment:
I intentionally moved the exception 'upstream', from the shell call to the setDeviceId call. Did this in several test cases for consistenty